src/UserService.Business/Commands/User/CreateUserCommand.cs

```diff
@@ -42,14 +42,61 @@ public class CreateUserCommand : ICreateUserCommand
         private readonly IAccessValidator _accessValidator;
 
         #region private methods
+
         private void ChangeUserDepartment(Guid departmentId, Guid userId, List<string> errors)
         {
-            // TODO add user department
+            string errorMessage = $"Сan't assign user {userId} to the department {departmentId}. Please try again later.";
+
+            try
+            {
+                var response = _rcDepartment.GetResponse<IOperationResult<bool>>(
+                    IChangeUserDepartmentRequest.CreateObj(userId, departmentId)).Result;
+                if (!response.Message.IsSuccess || !response.Message.Body)
+                {
+                    _logger.LogWarning(
```

Comment:
Why not use errorMessage  variable? And no use response.Message.Errors?

Reply:
this var uses $, but we don't need it in logging